code/modules/events/slaughterevent.dm

```diff
@@ -18,19 +18,8 @@
 	var/datum/mind/player_mind = new /datum/mind(key_of_slaughter)
 	player_mind.active = TRUE
 	var/list/spawn_locs = list()
-	for(var/thing in GLOB.landmarks_list)
-		var/obj/effect/landmark/L = thing
-		if(isturf(L.loc))
-			switch(L.name)
-				if("revenantspawn")
-					spawn_locs += L.loc
-	if(!spawn_locs) //If we can't find any revenant spawns, try the carp spawns
-		for(var/thing in GLOB.landmarks_list)
-			var/obj/effect/landmark/L = thing
-			if(isturf(L.loc))
-				switch(L.name)
-					if("carpspawn")
-						spawn_locs += L.loc
+	for(var/obj/effect/landmark/spawner/rev/L in GLOB.landmarks_list)
+		spawn_locs += get_turf(L)
 	if(!spawn_locs) //If we can't find either, just spawn the revenant at the player's location
```

Comment:
```suggestion
		if(!spawn_locs) //If we can't find a good place, just spawn the revenant at the player's location
```

Reply:
extra indentation would break stuff here, will apply the comment change manually